boards/launchxl/src/io.rs

```diff
@@ -0,0 +1,12 @@
+use core::fmt::Arguments;
+
+#[cfg(not(test))]
+#[lang = "panic_fmt"]
+#[no_mangle]
+pub unsafe extern "C" fn rust_begin_unwind(
+    _args: Arguments,
+    _file: &'static str,
+    _line: usize,
+) -> ! {
+    loop {}
+}
```

Comment:
I highly recommend you take a look at the other boards `io.rs` files and copy a little more of them in. Especially the LED blink pattern. It's nice to have a unique blink pattern that indicates "everything has crashed". It's fine if UART isn't working yet to not include that part.

Reply:
Sounds good! We have it included in a later commit, I'll cherry-pick it into this PR